tests/main/install-store-laaaarge/task.yaml

```diff
@@ -19,5 +19,5 @@ restore: |
 
 execute: |
     # test-snapd-tools is about 8k, tmpfs is 4k :-)
```

Comment:
The comment mentions test-snapd-tools. I am not sure I fully get the purpose of the test, but I think the idea was to have a very little space in /tmp, to make sure that downloads are not writing temporary files to /var/lib/snapd/partial instead of /tmp. So the purpose of the test is actually to pull the snap from the store.

Reply:
@bboozzoo thanks for the review, agree, and we are doing "snap install test-snapd-sh" which is pulling the snap from the store. The difference is that this snap is very light, so it is much faster to install/remove than the test-snapd-tools one.